.github/workflows/ihs-cisBuild.yml

```diff
@@ -0,0 +1,307 @@
+  # curl --verbose -XPOST -u "WASdev:${PERSONAL_ACCESS_TOKEN}" -H "Accept: application/vnd.github.everest-preview+json" -H "Content-Type: application/json" https://api.github.com/repos/${REPO_NAME}/actions/workflows/ihs-cisBuild.yml/dispatches --data '{"ref": "main"}'
+  repository_dispatch:
+    types: [integration-test-ihs-cis, integration-test-all]
+  # sample request
+  # PERSONAL_ACCESS_TOKEN=<GITHUB_PERSONAL_ACCESS_TOKEN>
+  # REPO_NAME=WASdev/azure.websphere-traditional.image
+  # curl --verbose -X POST https://api.github.com/repos/${REPO_NAME}/dispatches -H "Accept: application/vnd.github.everest-preview+json" -H "Authorization: token ${PERSONAL_ACCESS_TOKEN}" --data '{"event_type": "integration-test-ihs-cis"}'
+  # curl --verbose -X POST https://api.github.com/repos/${REPO_NAME}/dispatches -H "Accept: application/vnd.github.everest-preview+json" -H "Authorization: token ${PERSONAL_ACCESS_TOKEN}" --data '{"event_type": "integration-test-all"}'
+
+env:
+  repoName: azure.websphere-traditional.image
+  userName: ${{ secrets.USER_NAME }}
+  azureCredentials: ${{ secrets.AZURE_CREDENTIALS }}
+  entitledIbmUserId: ${{ secrets.ENTITLED_IBM_USER_ID }}
+  entitledIbmPassword: ${{ secrets.ENTITLED_IBM_USER_PWD }}
+  unEntitledIbmUserId: ${{ secrets.UNENTITLED_IBM_USER_ID }}
+  unEntitledIbmPassword: ${{ secrets.UNENTITLED_IBM_USER_PWD }}
+  vmAdminId: ${{ secrets.VM_ADMIN_ID }}
+  vmAdminPassword: ${{ secrets.VM_ADMIN_PASSWORD }}
+  testResourceGroup: ihsCISImageTest${{ github.run_id }}${{ github.run_number }}
+  vmName: vm${{ github.run_id }}${{ github.run_number }}
+  vhdStorageAccountName: storage${{ github.run_id }}${{ github.run_number }}
+  containerName: vhds
+  osDiskSnapshotName: osdisk-backup
+  dataDiskSnapshotName: datadisk-backup
+  osDiskVHDName: osdisk.vhd
+  dataDiskVHDName: datadisk.vhd
+  location: eastus
+  scriptLocation: https://raw.githubusercontent.com/${{ secrets.USER_NAME }}/azure.websphere-traditional.image/$GITHUB_REF_NAME/ihs-cis/test/
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      - name: Get versions of external dependencies
+        run: |
+          curl -Lo external-deps-versions.properties https://raw.githubusercontent.com/Azure/azure-javaee-iaas/main/external-deps-versions.properties
+          source external-deps-versions.properties
+          echo "azCliVersion=${AZ_CLI_VERSION}" >> $GITHUB_ENV
+          echo "refArmttk=${ARM_TTK_REFERENCE}" >> $GITHUB_ENV
+          echo "refJavaee=${AZURE_JAVAEE_IAAS_REFERENCE}" >> $GITHUB_ENV
+      - name: Set up JDK 8
+        uses: actions/setup-java@v2
+        with:
+          distribution: 'zulu'
+          java-version: '8'
```

Comment:
I will add a commit to change this to microsoft and 8. Zulu is no longer recommended.